src/Behat/Config/Profile.php

```diff
@@ -79,4 +110,125 @@ public function toArray(): array
     {
         return $this->settings;
     }
+
+    public function toPhpExpr(): Node\Expr
+    {
+        $profileObject =  $this->builderFactory->new(new FullyQualified(self::class));
+        $expr = $profileObject;
+
+        $this->addFormattersToExpr($expr);
+        $this->addFiltersToExpr($expr);
+        $this->addUnusedDefinitionsToExpr($expr);
+        $this->addExtensionsToExpr($expr);
+        $this->addSuitesToExpr($expr);
+
+        if (count($this->settings) === 0) {
+            $args = $this->builderFactory->args([$this->name]);
+        } else {
+            $args = $this->builderFactory->args([$this->name, $this->settings]);
+        }
+        $profileObject->args = $args;
+
+        return $expr;
+    }
+
+    private function addFormattersToExpr(Expr &$expr): void
+    {
+        if (isset($this->settings[self::FORMATTERS_SETTING])) {
+            foreach ($this->settings[self::FORMATTERS_SETTING] as $name => $formatterSettings) {
+                if ($formatterSettings === false) {
+                    $args = $this->builderFactory->args([$name]);
+                    $expr = $this->builderFactory->methodCall($expr, self::DISABLE_FORMATTER_FUNCTION, $args);
+                } else {
+                    if ($formatterSettings === null) {
+                        $formatterSettings = true;
+                    }
+                    if (isset($formatterSettings[ShowOutputOption::OPTION_NAME])) {
+                        $formatterSettings[ShowOutputOption::PARAMETER_NAME] =
+                            ShowOutputOption::from($formatterSettings[ShowOutputOption::OPTION_NAME]);
+                        unset($formatterSettings[ShowOutputOption::OPTION_NAME]);
+                    }
+
+                    $formatter = match($name) {
+                        PrettyFormatter::NAME => $formatterSettings === true ? new PrettyFormatter() : new PrettyFormatter(...$formatterSettings),
```

Comment:
We create a new formatter object with all passed options, including base options like `output_path`